webui/tests/bin/e2e-tests.py

```diff
@@ -87,94 +98,45 @@ def _cypress_arguments(cypress_configs, config, use_docker):
     return args
 
 
-def container_exists(name):
-    return any(filter(lambda container: container.name == name, client.containers.list(all=True)))
-
-
-def clean_up_cypress(config):
-    cypress_name = _cypress_container_name(config)
-    if container_exists(cypress_name):
-        # ensure that the cypress container is stopped and removed
-        run(["docker", "container", "rm", "-f", cypress_name], config)
-
-
 def run_e2e_tests(config):
-    base_url_config = f"baseUrl=http://localhost:{config['INTEGRATIONS_HOST_PORT']}"
-    cypress_arguments = _cypress_arguments([base_url_config], config, False)
-
-    command = ["yarn", "--cwd", str(tests_dir), "run", "cypress", "run", *cypress_arguments]
-
-    run(
-        command, config,
-    )
-
-
-def docker_run_e2e_tests(config):
-    cluster_name = config["CLUSTER_NAME"]
-    master_name = cluster_name + "_determined-master_1"
-    network_name = cluster_name + "_default"
-    cypress_name = _cypress_container_name(config)
-
-    base_url_config = f"baseUrl=http://{master_name}:8080"
-    cypress_config = [base_url_config]
-    cypress_arguments = _cypress_arguments(cypress_config, config, True)
-
+    """ depends on:
+    1. a brand new, exclusive cluster at config['DET_MASTER']
+    2. pre_e2e_tests() to have seeded that cluster recently* """
+    cypress_arguments = _cypress_arguments([], config)
     command = [
-        "docker",
+        "yarn",
+        "--cwd",
+        str(tests_dir),
+        "run",
+        "cypress",
```

Comment:
this would probably fail for setup without `yarn`. `npm`/`npx` equivalent?

Reply:
oh yeah good catch. we shouldn't have `yarn` there but we can just depend on npm/npx as that's project dependency right?